src/global-styles/normalize.css

```diff
@@ -0,0 +1,354 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
we shouldn't need this, we're already using `postcss-normalize`. Let's figure out if we can make that do what we need.

Reply:
remove